cluster/src/main/scala/org/apache/spark/memory/SnappyStorageEvictor.scala

```diff
@@ -66,7 +66,11 @@ class SnappyStorageEvictor extends Logging{
   }
 
   @throws(classOf[Exception])
-  def evictRegionData(bytesRequired: Long): Long = {
+  def evictRegionData(bytesRequired: Long, memoryMode: MemoryMode): Long = {
+    // @TODO use when regions can be stored in offheap
+    if (memoryMode.equals(MemoryMode.OFF_HEAP)){
+      return 0L
```

Comment:
Does this mean off-heap will skip the evictor? I guess this needs to implement the separate eviction logic for off-heap as we discussed.